src/components/Image/Image.visual.tsx

```diff
@@ -60,6 +60,21 @@ visualize('Image', () => {
         />
       ));
 
+      story('with aspectRatio', () => {
```

Comment:
After we decide on the valid variations I'll add additional tests representing more combinations (aspectRatio with/without dimensions)